after/conf/config.yaml

```diff
@@ -1,9 +1,7 @@
-defaults:
-  - files: mnist
-  - _self_
+files: files/mnist.yaml
 paths:
   log: ./runs
-  data: ${hydra:runtime.cwd}/../data/raw
+  data: ../data/raw
```

Comment:
jsonargparse is not intended to be a framework that automatically creates directories for each run. Something like this would need to be built on top of it. For the case of pytorch this has already been done in pytorch-lighting, but it does much more than hydra. People are able to create CLIs with minimal code, in simple cases it is just one line of code. And they don't need to learn about a complex framework like hydra or jsonargparse to implement and use it. See https://pytorch-lightning.readthedocs.io/en/latest/common/lightning_cli.html